### PR TITLE
fix: remove product colors and update example brand colors

### PR DESCRIPTION
**Pull request by jzempel**
- [ ] **BREAKING CHANGE?** 

## Description

The main features of this PR:

- Apply `DEFAULT_THEME.palette` rather than `PALETTE` (missed this on initial review). Ideally, we wouldn't have `product` values in Garden's design tokens
- Update `answer-bot` colors per brand guidelines

## Detail

In addition, this PR provides the following:

- An important fix to add LICENSE.md to the published npm package
- Convenience root package scripts so that this repo mirrors the basic structure of other Garden repositories
  - Run `yarn` and everything is ready-to-go
  - A simple `yarn start` boots up the demo
- Root file `format` and `lint` coverage

### CODE REVIEW

File: plugin/src/__snapshots__/theme.spec.ts.snap [hunk 1]
```diff
@@ -108,17 +108,6 @@ Object {
...
-      "chat": "#f79a3e",
-      "connect": "#eb6651",
-      "explore": "#30aabc",
-      "gather": "#e7afa2",
-      "guide": "#eb4962",
-      "message": "#37b8af",
-      "sell": "#d4ae5e",
-      "support": "#78a300",
-      "talk": "#efc93d",
-    },
```
Comment (jzempel) [inline plugin/src/__snapshots__/theme.spec.ts.snap:121]: The removal of this key might warrant some discussion. We never want folks to use these product color values for theming (which is why they are removed from the `DEFAULT_THEME` in `@zendeskgarden/react-theming`). Furthermore, I'm considering this to be a patch – as the initial addition as design tokens could be considered unintentional – but I could be convinced to re-characterize this PR as a "BREAKING CHANGE".

Comment (austin94) [reply plugin/src/__snapshots__/theme.spec.ts.snap:121]: I would like to discuss this removal more.

> We never want folks to use these product color values for theming

I agree with this in relation to dynamic theming in `react-components`, but these values could be used often throughout a codebase which uses `tailwindcss`. If we were to remove these products would hardcode the values and no longer receive updates as values are updated.

Comment (jzempel) [reply plugin/src/__snapshots__/theme.spec.ts.snap:121]: Confirmed by design: product colors are only for product and only used for 1️⃣ purpose – the logo. Never to be used in the context of design tokens.

**Review by austin94 — APPROVED** [review]
LGTM as a `patch` release. If this was never intended to be included we should be ok.

**Review by hzhu — APPROVED** [review]

**Pull request merged**